test/integration/fixtures/config.js

```diff
@@ -1,30 +1,48 @@
-if (typeof process.env.DC_ADDR !== 'string') {
-  console.error('Missing DC_ADDR environment variable!')
-  process.exit(1)
-}
-
-if (typeof process.env.DC_MAIL_PW !== 'string') {
-  console.error('Missing DC_MAIL_PW environment variable!')
-  process.exit(1)
-}
+const conf = require('rc')('DC', {})
 
-const getConfig = () => {
-  return {
-    notifications: true,
-    enterKeySends: true,
-    showNotificationContent: true,
-    locale: 'en',
+if (conf.account1 === undefined || conf.account1.credentials === undefined) {
+  if (typeof process.env.DC_ADDR !== 'string') {
+    console.error('Missing DC_ADDR environment variable!')
+    process.exit(1)
+  }
+  if (typeof process.env.DC_MAIL_PW !== 'string') {
+    console.error('Missing DC_MAIL_PW environment variable!')
+    process.exit(1)
+  }
+  if (typeof process.env.DC_MAIL_PW1 !== 'string') {
+    console.error('Missing DC_MAIL_PW1 environment variable!')
```

Comment:
Why do we need PW, PW1 and PW2 variables?

Reply:
I would like to have tests including starting a one-to-one chat and a group chat, so we would need 3 accounts.
I will provide a sample how to configure that for local tests with a .DCrc file including the needed configuration as json.

Reply:
But you didn't introduce this test yet in this pr? I would prefer to have this pr only focus on switching to mocha if possible, even if i like testing group chats quite a lot! But for this I think it would be a better approach to implement the usage of the testrun.org burner account creation script. This way you only need a api token and everything else is getting created on the fly.